unfetter-discover/unfetter-ui
# fix spelling, lint

Language is a tool.  Smash it over something like a hammer.  If it fixes the problem great, if it breaks the hammer; even by my own fault, throw it away and get a new one.

requires, https://github.com/unfetter-discover/unfetter/pull/643

## To Test
* pull both projects
* open mongo shell, delete reports `db.getCollection('stix').remove({ 'stix.type': 'report' })`
* restart stack
* visit threat dashboard pages, https://localhost/#/threat-dashboard
* verify boundaries spelling is fixed
